demosplan/DemosPlanCoreBundle/Controller/Statement/DemosPlanStatementController.php

```diff
@@ -70,23 +67,8 @@
 use demosplan\DemosPlanUserBundle\Logic\CurrentUserService;
 use demosplan\DemosPlanUserBundle\Logic\OrgaHandler;
 use demosplan\DemosPlanUserBundle\Logic\UserService;
-
-use const ENT_QUOTES;
-
 use Exception;
-
-use function explode;
-use function html_entity_decode;
-use function implode;
```

Comment:
Why was the use statements of the functions removed? In other files they still remaining. The calls to the functions are supplemented by a "\\". According to what rule do we use it like this?